extensions/jackson/deployment/src/main/resources/application-fail-on-unknown-properties.properties

```diff
@@ -0,0 +1 @@
+quarkus.jackson.fail-on-unknown-properties=true
```

Comment:
Shoudn't this file be inside `src/test/resources` instead ?

Reply:
Ah, yes sure.